src/plt-elf.c

```diff
@@ -195,9 +195,8 @@ plt_lib plt_get_lib(plt_ctx ctx, const char *name)
             if (!strcmp(name, libname))
                 return lm;
         } else if (sel == PLT_SEL_SYM) {
-            plt_fn **sym = plt_get_offset(ctx, lm, val);
-            if (sym)
-                return lm;
+          if (get_offsets(ctx, lm, val, NULL) > 0)
```

Comment:
nit: 4-space indent

Reply:
Done in https://github.com/Snaipe/Mimick/pull/10/commits/a275442693bc5e4c8f8ba537313ed0d4834c0ca0.